modules/components/forms/text_input.cjsx

```diff
@@ -21,6 +21,7 @@ module.exports = class TextInput extends RestFormElement
 
   @defaultProps =
     inputType: 'text'
+    value:     ''
```

Comment:
We should move this onto `rest_form_element` to ensure we aren't accidentally switching between controlled and uncontrolled inputs.